checkbox-support/checkbox_support/vendor/beacontools/scanner.py

```diff
@@ -46,6 +46,7 @@ class HCIVersion(IntEnum):
     BT_CORE_SPEC_5_1 = 10
     BT_CORE_SPEC_5_2 = 11
     BT_CORE_SPEC_5_3 = 12
+    BT_CORE_SPEC_5_4 = 13
```

Comment:
Could you please help me find a place where this enumeration is used?

Reply:
This is used in `def set_scan_parameters` of `scanner.py` to set different parameters.

Reply:
it's an inverse mapping, this is why me grepping didn't show anything... Now I get it, thank you.

Reply:
I'm marking this conversation as resolved then